pkg/pillar/cmd/downloader/syncop.go

```diff
@@ -337,7 +337,7 @@ func getDatastoreCredential(ctx *downloaderContext,
 			decBlock.DsPassword = dst.Password
 			return decBlock, nil
 		}
-		log.Infof("%s, cipherblock decryption successful\n", dst.Key())
+		log.Infof("%s, datastore config cipherblock decryption successful\n", dst.Key())
```

Comment:
You can get rid of the \n at the end going forward.. Whenever to touch a log statement next time :-) You don;t need that..